Lib/idlelib/query.py

```diff
@@ -83,7 +84,7 @@ def __init__(self, parent, title, message, *, text0='', used_names={},
             self.deiconify()  # Unhide now that geometry set.
             self.wait_window()
 
-    def create_widgets(self):  # Call from override, if any.
+    def create_widgets(self, ok_text='OK'):  # Call from override, if any.
```

Comment:
I made this configurable so that the 'Custom Run' dialog could change the `OK` to `Run`.

Reply:
Sounds good!  🙂